app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from typing import List
```

Comment:
Consider separating different groups of imports by an empty string, here you use two groups - built-in package import and import of local files.